pyfstat/make_sfts.py

```diff
@@ -156,19 +155,19 @@ def _get_setup_from_noiseSFTs(self):
 
             IFOs.append(ifo_name)
             tstart.append(this_start_time)
-            tend.append(this_end_time)
+            tend.append(this_end_time + this_Tsft)
             Tsft.append(this_Tsft)
 
         # Get the "overall" values of the search
-        self.tstart = min(tstart)
-        self.tend = max(tend)
-        self.duration = self.tend - self.tstart + this_Tsft
-        self.IFOs = ",".join(['"{}"'.format(d) for d in IFOs])
         Tsft = np.unique(Tsft)
         if len(Tsft) != 1:
             raise ValueError("SFTs contain different basetimes: {}".format(Tsft))
         else:
             self.Tsft = Tsft[0]
+        self.tstart = min(tstart)
+        self.tend = max(tend)
+        self.duration = self.tend - self.tstart + self.Tsft
```

Comment:
...then here the `Tsft` is doubled up?
```suggestion
        self.duration = self.tend - self.tstart
```

Reply:
Yes, definitely